### ISSUE TITLE
Antag roles being "unfound"

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#15988]
(WEB REPORT BY: bobsmerf REMOTE: 172.93.109.202:7777)
# Revision
38493f043289c9006d6812d133a7281d286c2965

# Description
Every time I go to save my preferences it says that the antag roles can not be found. As a result one can not be an antag.

# Steps to Reproduce
Save preferences in character select.

# What you Expected
To be able to change my antag preferences.
# What Actually Happened
Preferences were unfound.

**Comment by Kitfox88** [comment]
Happening to me too. I figured it was just some shiftily implemented whitelist though.

**Comment by MadmanMartian** [comment]
Been looking into this, and seems SaveRoles just isn't passed antag preferences, or they're being read wrong

**Comment by MadmanMartian** [comment]
yeah, SetRoles is passed href_list, which only contains _src_ and _preferences_, not any of the roles
